src/components/KnockFeedProvider/KnockFeedProvider.tsx

```diff
@@ -1,15 +1,15 @@
 import * as React from "react";
 import Knock, { Feed, FeedStoreState } from "@knocklabs/client";
 import create, { UseStore } from "zustand";
-import { FilterStatus, ColorMode } from "../../constants";
+import { ColorMode } from "../../constants";
 import { KnockFeedContainer } from "./KnockFeedContainer";
+import useAuthenticatedKnockClient from "../../hooks/useAuthenticatedKnockClient";
+import useFeedClient from "../../hooks/useFeedClient";
 
 export interface KnockFeedProviderState {
   knock: Knock;
   feedClient: Feed;
   useFeedStore: UseStore<FeedStoreState>;
-  status: FilterStatus;
```

Comment:
Removed these as they never really made any sense in this component. Moved down to `NotificationFeed` instead.